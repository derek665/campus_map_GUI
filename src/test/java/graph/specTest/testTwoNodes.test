# test when graph is constructed and add two nodes to it, and print the nodes

# Create a graph
CreateGraph twoNodes

# Add 2 nodes to graph
AddNode twoNodes n1
AddNode twoNodes n2

# Print the nodes in graph
ListNodes twoNodes