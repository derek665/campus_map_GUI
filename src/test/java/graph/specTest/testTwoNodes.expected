# test when graph is constructed and add two nodes to it, and print the nodes

# Create a graph
created graph twoNodes

# Add 2 nodes to graph
added node n1 to twoNodes
added node n2 to twoNodes

# Print the nodes in graph
twoNodes contains: n1 n2